### ISSUE TITLE
unable to parse XXX value=0i': invalid number

### ISSUE THREAD

**Issue by jonaz** [influxdb/influxdb#3606]
influxdb server version 0.9.2

Im using the Go client library revision 9ece632791a4fb62be81d826a77b9937fe67c3ce and recently i've been unable to insert integers. 
Not sure if the clients appends the i and thats the problem or something else. 

When i try appending "i" in the command line i get the same error as when using the Go library.  My code using the library is here: https://github.com/stampzilla/stampzilla-go/blob/master/nodes/stampzilla-server/influxdb.go#L74

The error logged on line 76 is:
```
influxdb.go:76 - unable to parse '99da729e-2563-40be-848c-32427b140873_Node_State_Devices_1_State_On value=0i 1439206597145923541': invalid number
```

Some debugging with CLI showing its a integer measurements. 
```
> insert 99da729e-2563-40be-848c-32427b140873_Node_State_Devices_1_State_On value=0i
ERR: unable to parse ' 99da729e-2563-40be-848c-32427b140873_Node_State_Devices_1_State_On value=0i': invalid number

> insert 99da729e-2563-40be-848c-32427b140873_Node_State_Devices_1_State_On value=1i
ERR: unable to parse ' 99da729e-2563-40be-848c-32427b140873_Node_State_Devices_1_State_On value=1i': invalid number

> insert 99da729e-2563-40be-848c-32427b140873_Node_State_Devices_1_State_On value=1
> insert 99da729e-2563-40be-848c-32427b140873_Node_State_Devices_1_State_On value=0
> insert 99da729e-2563-40be-848c-32427b140873_Node_State_Devices_1_State_On value=1.0
ERR: write failed: field type conflict: input field "value" on measurement "99da729e-2563-40be-848c-32427b140873_Node_State_Devices_1_State_On" is type float64, already exists as type integer

> insert 99da729e-2563-40be-848c-32427b140873_Node_State_Devices_1_State_On value=1
> 
```

**Comment by maxekman** [comment]
I get the same error when running the current client library against 0.9.2. In this case it was a uint64 from the telegraf/system/memory.go plugin.

```
unable to parse 'mem_total,host=localhost value=2050420000i': invalid number
```

**Comment by maxekman** [comment]
The protocol change was added in this commit, which I presume is post-0.9.2: https://github.com/influxdb/influxdb/commit/0b5eb09181c9b6fed3218a4f540201adbc3e5310

**Comment by juddgaddie** [comment]
You need to run that version of the go client against  the latest nightly build of Influxdb or downgrade the go client to a version that did not inert i for integers.

**Comment by maxekman** [comment]
I can confirm that checking out 0.9.2 in $GOPATH/src/influxdb/influxdb works.

**Comment by jonaz** [comment]
OK, closing this then!

**Comment by felixbarny** [comment]
I still have this issue with v0.9.2.1. I'm using influxdb with docker-compose and tutum/influxdb

```
$ curl -i -XPOST 'http://192.168.99.100:8086/write?db=stagemonitor' --data-binary 'cpu value=10i'
HTTP/1.1 400 Bad Request
Request-Id: 1eb5c813-4432-11e5-801e-000000000000
X-Influxdb-Version: 0.9.2.1
Date: Sun, 16 Aug 2015 16:16:15 GMT
Content-Length: 48
Content-Type: text/plain; charset=utf-8

unable to parse 'cpu value=10i': invalid number
```

**Comment by juddgaddie** [comment]
You need to upgrade to a nightly build or stop using the i suffix.

**Comment by felixbarny** [comment]
Ok. I assumed it should have been fixed in 0.9.2.1 based off the comment
> OK, can also confirm checking out tag v0.9.2.1 works.

Probably @jonaz ment branch, not tag?

**Comment by jonaz** [comment]
I means using the Go client version in tag v0.9.2.1 with server version 0.9.2
